basedosdados_api/api/v1/models.py

```diff
@@ -370,8 +372,8 @@ class Table(BdmModel):
     partner_organization = models.ForeignKey(
         "Organization", on_delete=models.CASCADE, related_name="partner_tables"
     )
-    update_frequency = models.ForeignKey(
-        "UpdateFrequency", on_delete=models.CASCADE, related_name="tables"
+    update = models.ForeignKey(
+        "Update", on_delete=models.CASCADE, related_name="tables"
```

Comment:
Também está obrigatório. Teremos como popular isso na migração?

Reply:
Como faz para campos foreign_key não serem obrigatórios? Tem que transformar em many to many?